### PR TITLE
Remove srcset attribute if present to actually show hi-res image when zoomed-in

### PR DESCRIPTION
**Pull request by mahish**
…and put it back when zoomed-out.

Another PR:) If `srcset` is present on `` tag, then the hi-res image is not rendered. This PR fix this issue by temporarily removing `srcset` while zooming-in.

How to deal with `<picture>` and `<source>` is for another story tho…

### CODE REVIEW

**Comment by kingdido999** [comment]
Thanks for the PR! Would you mind do a quick demo on this `srcset` issue by forking this [template](https://codepen.io/kingdido999/pen/rpYrKV)?

**Comment by mahish** [comment]
Sure, here: https://codepen.io/mahish/pen/NyYPpG

File: src/modules/target.js [hunk 1]
```diff
@@ -101,7 +105,11 @@ export default {
       // Add delay to prevent Firefox from flickering
       setTimeout(
         function updateSrc () {
+          if (this.srcset) {
+            this.el.setAttribute('srcset', '')
```
Comment (kingdido999) [inline src/modules/target.js]: Maybe [Element.removeAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/removeAttribute) is more appropriate here?

File: src/modules/target.js [hunk 2]
```diff
@@ -93,6 +94,9 @@ export default {
 
       // Force compute the hi-res image in DOM to prevent
       // image flickering while updating src
+      if (this.srcset) {
+        temp.removeAttribute('srcset')
```
Comment (kingdido999) [inline src/modules/target.js]: Can we remove `srcset` from `this.el` before cloning it to`temp` (before line 93)? Just so we don't need to do the double work on `temp`.

**Comment by mahish** [comment]
@kingdido999 Yea, much better.

File: src/modules/target.js (hunk 2 above)
Comment (mahish) [reply src/modules/target.js]: Yea, much better.

**Pull request merged**